lib/ssl/src/tls_server_session_ticket.erl

```diff
@@ -385,10 +385,10 @@ stateless_living_ticket(0, _, _, _, _) ->
 stateless_living_ticket(ObfAge, TicketAgeAdd, Lifetime, Timestamp, Window) ->
     ReportedAge = ObfAge - TicketAgeAdd,
     RealAge = erlang:system_time(second) - Timestamp,
-    (ReportedAge =< Lifetime)
+    (ReportedAge =< Lifetime * 1000)
```

Comment:
I think it doesn't matter, since this is stored in the encrypted ticket and only visible to the server, so the unit of the time stamp generated in line 329 just needs to match the time stamp in line 387, when computing the real ticket age. These are both in seconds and the elapsed time is compared to the `Lifetime`, which is also in seconds.